client/src/components/Header/index.tsx

```diff
@@ -43,9 +42,7 @@ export class Header extends React.Component<
     if (
```

Comment:
Since it's not so obvious why this is necessary, could you add a comment explaining why the `searchBarRef` needs checking here?